benchalerts/parse_conbench.py

```diff
@@ -46,35 +85,58 @@ def benchmarks_with_z_regressions(
 def regression_summary(
     comparisons: List[RunComparison], warn_if_baseline_isnt_parent: bool
 ) -> str:
-    """Generate a Markdown summary of what happened regarding regressions."""
+    """Generate a Markdown summary of what happened regarding errors and regressions."""
     sha = comparisons[0].contender_info["commit"]["sha"][:8]
+    errors = benchmarks_with_errors(comparisons)
+    regressions = benchmarks_with_z_regressions(comparisons)
+    summary = ""
+
+    if errors:
+        summary += _clean(
+            """
+            ## Benchmarks with errors
+
+            These are errors that were caught while running the benchmarks. You can
+            click the link next to each case to go to the Conbench entry for that
+            benchmark, which might have more information about what the error was.
+            """
+        )
+        previous_run_id = ""
+        for run_id, run_url, name, error_url in errors:
+            if run_id != previous_run_id:
+                summary += f"\n\n- Run ID [{run_id}]({run_url})"
```

Comment:
Very minor + mostly style: Would it be possible to have this be a date time or the run name or something like that instead of the SHA? (we can also have the sha if we want, but maybe a short version of it?) The Run IDs look a little unweildy | I worry folks might glaze over seeing so many shas like that.

Reply:
Yes, datetime is a great idea. I'll throw in the run reason too.